integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -534,6 +534,7 @@ private static boolean createRbacApiObjectsForWLDFScript(String domainNamespace,
               .addResourcesItem("domains")
               .addVerbsItem("get")
               .addVerbsItem("list")
+              .addVerbsItem("patch")
```

Comment:
We are adding a required role verb (this seems correct), but reminds me that we need to document this requirement in the scaling and/or RBAC documentation. Similarly, we need to document how to create the correct roles and role bindings so that the service account used with the scaling API has the necessary permissions.

Reply:
Updated documentation in /userguide/managing-operators/using-the-operator/using-helm.md, /userguide/managing-operators/using-the-operator/the-rest-api.md, /userguide/managing-domains/domain-lifecycle/scaling.md, and /kubernetes/charts/weblogic-operator/values.yaml